setup/git/010_TestSetup.rb

```diff
@@ -41,34 +42,18 @@
           # !/usr/bin/env ruby
           # the problem is a Cygwin style path is passed to the interpreter and this can't be modified:
           # http://cygwin.1069669.n5.nabble.com/Pass-windows-style-paths-to-the-interpreter-from-the-shebang-line-td43870.html
-          on host, "cd #{puppet_dir} && cmd.exe /c \"bundle install --system --binstubs '#{binstubs_dir}'\""
-          # puppet.bat isn't written by Bundler, but facter.bat is - copy this generic file
-          on host, "cd #{host['puppetbindir']} && test -f ./puppet.bat || cp ./facter.bat ./puppet.bat"
-          # to access gem / facter / puppet / bundle / irb with Cygwin generally requires aliases
-          # so that commands in /usr/bin are overridden and the binstub wrappers won't run inside Cygwin
-          # but rather will execute as batch files through cmd.exe
-          # without being overridden, Cygwin reads the shebang and causes errors like:
-          # C:\cygwin64\bin\ruby.exe: No such file or directory -- /usr/bin/puppet (LoadError)
-          # NOTE /usr/bin/puppet is a Cygwin style path that our custom Ruby build
-          # does not understand - it expects a standard Windows path like c:\cygwin64\bin\puppet
+          on host, "cd #{puppet_dir} && #{bundle_command(host)} install --system --binstubs '#{binstubs_dir}'"
 
-          # a workaround in interactive SSH is to add aliases to local session / .bashrc:
-          #   on host, "echo \"alias puppet='C:/\\cygwin64/\\bin/\\puppet.bat'\" >> ~/.bashrc"
-          # note that this WILL NOT impact Beaker runs though
-          puppet_bundler_install_dir = on(host, "cd #{puppet_dir} && cmd.exe /c bundle show puppet").stdout.chomp
-        when /el-7/
-          create_remote_file(host, "#{puppet_dir}/Gemfile", gemfile_contents + "gem 'json'\n")
```

Comment:
I couldn't find any detail around why we needed to add the json gem for el7, so I removed it, and nothing broke...

Reply:
👍 The ruby package provided by the distro oddly didn't have json, so we had to add it back in. But now that we're using the runtime, we don't have to worry about that nonsense.

Reply:
sweetness